sklearn/metrics/tests/test_regression.py

```diff
@@ -75,4 +80,70 @@ def test__check_reg_targets():
                 assert_array_equal(y_check1, y1)
                 assert_array_equal(y_check2, y2)
         else:
-            assert_raises(ValueError, _check_reg_targets, y1, y2)
+            assert_raises(ValueError, _check_reg_targets, y1, y2, None)
+
+
+def test_regression_multioutput_array():
+    y_true = [[1, 2], [2.5, -1], [4.5, 3], [5, 7]]
+    y_pred = [[1, 1], [2, -1], [5, 4], [5, 6.5]]
+
+    mse = mean_squared_error(y_true, y_pred, multioutput='raw_scores')
+    mae = mean_absolute_error(y_true, y_pred, multioutput='raw_scores')
+    r = r2_score(y_true, y_pred, multioutput='raw_scores')
+    evs = explained_variance_score(y_true, y_pred, multioutput='raw_scores')
+
+    assert_array_equal(mse, np.array([0.125, 0.5625]))
+    assert_array_equal(mae, np.array([0.25, 0.625]))
```

Comment:
Can you use array_almost_equal? Otherwise, this might create weird failures on some combination of OS/hardware.